web/modules/custom/do_feed/config/install/views.view.feed.yml

```
@@ -0,0 +1,261 @@
+tag: default
+base_table: node_field_data
+base_field: nid
+display:
+  default:
+    id: default
+    display_title: Default
+    display_plugin: default
+    position: 0
+    display_options:
+      title: 'Blog RSS Feed'
+      fields: {  }
```

YAMLlint (1.38.0)

[error] too many spaces inside empty braces

(braces)